src/controllers/requestController.js

```diff
@@ -90,23 +91,35 @@ export default class requestController {
 
   static async updateRequest(req, res) {
     const { id } = req.request;
+
     try {
       await models.requests.update(req.body, {
         where: { id }, returning: true, raw: true,
       });
+
       const destination = req.body.destinations;
+      const { lineManager } = req.user.payload;
+      const APP_URL_BACKEND = `${req.protocol}://${req.headers.host}`;
+
       destination.forEach(async element => {
         await models.destinations.update(element, {
-          where: {
-            [Op.and]: [{ requestId: id }, { id: element.id }]
-          },
+          where: { [Op.and]: [{ requestId: id }, { id: element.id }] },
         });
       });
+
       const request = await allSearch({ id });
+      const requestData = request[0].dataValues;
+
+      const notification = await notifBuilder(
+        requestData,
+        lineManager,
+        `request ${id} has been edited . Click here to view: ${APP_URL_BACKEND}/api/v1/requests/${id}.`
```

Comment:
Kindly edit the message into `A request has been edited...` or `The request of id: ${id} has been edited...`